install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -200,5 +195,11 @@ DELETE FROM `PREFIX_configuration` WHERE `name` IN (
   'SHOP_LOGO_MOBILE_WIDTH');
 
 ALTER TABLE `PREFIX_tab` ADD `icon` varchar(32) DEFAULT '';
+ALTER TABLE `PREFIX_profile_lang` ADD UNIQUE KEY (`name`);
```

Comment:
Same comment here, changes brought in this file won't applied for merchants already in PrestaShop 1.7.0.x

They must be moved in `install-dev/upgrade/sql/1.7.1.0.sql`